lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -459,6 +460,15 @@ def non_fractional_currency?(currency)
       def emv_payment?(payment)
         payment.respond_to?(:emv?) && payment.emv?
       end
+
+      def error_code_from(response)
+        code = response['error']['code'] 
+        decline_code = response['error']['decline_code'] if code == 'card_declined'
```

Comment:
I think when we originally built these we copied them off of what Stripe had; the odds of this ever going out of sync with what they have are pretty low. That said - good arguments here for keeping it as a separate string, so I say keep it.